build.gradle

```diff
@@ -21,16 +23,29 @@ buildscript {
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
         classpath 'gradle.plugin.org.spongepowered:spongegradle:0.8.1'
         classpath 'org.spongepowered:mixingradle:0.4-SNAPSHOT'
+        classpath mcTesterDep
     }
 }
 
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.2.51'
+}
+
+
+repositories {
+    maven { url 'https://jitpack.io' }
```

Comment:
Should be deployed to a proper repository before being merged.

Reply:
Or aaron cannot afford a maven repo